pkg/cmd/pipelinerun/delete_test.go

```diff
@@ -338,7 +338,7 @@ func TestPipelineRunDelete(t *testing.T) {
 			input:       seeds[7].pipelineClient,
 			inputStream: nil,
 			wantError:   true,
-			want:        "--keep-since option can only be used with --all",
+			want:        "must provide PipelineRun name(s) or use --pipeline flag or --all flag to use delete",
```

Comment:
why should this fail 
`"tkn ", "pr", "delete", "-f", "--keep-since", "60", "-n", "ns"`
User may want to delete all the pipelinrun older than 60 mins.
Also this would be similar to 
`"tkn ", "pr", "delete", "-f", "--keep", "x", "-n", "ns"`

Reply:
yeah that's true. In that case user may have to run the command `tkn pr delete -f --keep-since 60 --all -n ns"